src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.tsx

```diff
@@ -112,7 +115,7 @@ export class ConnectedFilterControl extends Component<
     const moreFiltersButtonContainerClassname = classNames(
       styles.MoreFiltersButtonContainer,
       actionsToRender.length === 0 &&
-        newDesignLanguage &&
+        (newDesignLanguage || queryFieldHidden) &&
```

Comment:
Why do we need this?

Reply:
If the query field is hidden AND only the `More Filters` button is showing, we have to round the left borders.
Previously we didn't have to do that because the in the old design, the `More Filters` button would have been up against the text field.

**With Query Field Hidden:**
<img width="451" alt="image" src="https://user-images.githubusercontent.com/72987032/101515828-caf0ab80-394c-11eb-88ff-7c0a8adb4788.png">



**Without Query Field Hidden:**
<img width="451" alt="image" src="https://user-images.githubusercontent.com/72987032/101515933-e360c600-394c-11eb-9d20-d5c7d58efe9f.png">